private/pin.sh

```diff
@@ -19,7 +21,7 @@ cat <<EOF
 maven_install(
     artifacts = # ...,
     repositories = # ...,
-    maven_install_json = "//:{repository_name}_install.json",
+    maven_install_json = "@${workspace_name}//:{repository_name}_install.json",
```

Comment:
`@$workspace_name` instead of `@${workspace_name}` because `{..}` is used for the `repository_ctx.template` format.

Reply:
Thanks.